pkg/verify/verify.go

```diff
@@ -47,25 +58,37 @@ var validReleaseDigest = regexp.MustCompile(`^[a-zA-Z0-9:]+$`)
 // digest - all verifiers must have at least one valid signature attesting the release
 // digest. If any failure occurs the caller should assume the content is unverified.
 type ReleaseVerifier struct {
-	verifiers     map[string]openpgp.EntityList
-	stores        []*url.URL
+	verifiers map[string]openpgp.EntityList
+
+	locations     []*url.URL
```

Comment:
Why keep the URI slice?  Can't we create just create `SignatureStore`s for these in `NewReleaseVerifier`?  Or drop `locations` from `NewReleaseVerifier` and have `NewFromConfigMapData` (or whatever we end up calling that) create the stores for each URI and call `WithStores`.

Reply:
I've pushed up [a verify-store-api][1] branch (currently at 0cc3e41c1) to do this, polish the APIs up a bit more (e.g. using a callback so sig-store consumers can cancel sig iteration), and decompose things into more, smaller subpackages.  Still need to go through and rework the persistence logic to use [my preferred add/evict callbacks][2], but the API changes I already have in my branch are fairly orthogonal to that, so they should be ready for review now.

[1]: https://github.com/wking/cluster-version-operator/tree/verify-store-api
[2]: https://github.com/openshift/cluster-version-operator/pull/279#discussion_r359088967

Reply:
> I've pushed up a verify-store-api branch...

Spun this out into the parallel #294.

Reply:
Picking up this change, or not, does not appear to be hugely impactful especially if we consider that much of this code may be OBE post 4.4 if we deploy a local cincy. Since this is holding up other 4.4 changes we need to make a decision. I would expect that this new code will be owned by the OTA team. Can we therefore move the final approval to that team and @LalatenduMohanty